matter/wifi/rs911x/wfx_rsi_host.c

```diff
@@ -34,7 +34,7 @@ sl_status_t wfx_wifi_start(void)
   /*
      * Create the driver task
      */
-  if (xTaskCreate(wfx_rsi_task, "wfx_rsi", WFX_RSI_TASK_SZ, NULL, 1, &wfx_rsi.wlan_task) != pdPASS) {
+  if (xTaskCreate(wfx_rsi_task, "wfx_rsi", WFX_RSI_TASK_SZ, NULL, TASK_PRIORITY_1, &wfx_rsi.wlan_task) != pdPASS) {
```

Comment:
Similar commment about the task priority name but I guess if the same TASK_PRIORITY_1 are reused maybe it should be a generic name like
`WFX_TASKS_PRIORITY`